### ISSUE TITLE
New content- telehealth content prototype

### ISSUE THREAD

**Issue by abquirarte** [cagov/covid19#79]
### Background
Californians need to be able to access healthcare without going into the doctor. Telehealth is a relatively new term for people. We want to be able to inform the public that is is an option for them during this public health crisis. 

The prototype will depend on provider data. A person should be able to look up their provider and find whether they offer telehealth options. 

### Content lead
Liz 

### Timeline
Wed/Thurs 

#### suggested original content and request
>Telemedicine is the safest way right now to talk to a nurse or doctor if you are not seriously ill or are concerned you may have the coronavirus

>First please contact your doctor’s office for advice on what to do, or to have a telemedicine visit with your doctor if they are able to do that

>If you are unable to get through to your doctor, or they cannot do a telemedicine visit, you can access a nurse or doctor on line or by telephone through your health plan.  They can help you, and direct you to where you should go if you need to be seen in person.

>Please see the list of health plans below with the link to their website so that you can access your health plans nurse advice line or doctor on line service if available

>Then list health plans alphabetically and have a  link to their home page.  The associations are working with all of the Medi-Cal plans to put COVID-19 resources and the access to their nurse advice line (all have one) and/or doctor on line service (some have one) prominently visible on their website home page or at most one click away.  Some already do.  Commercial and Medicare plans could do the same.

>DHCS will not have Medi-Cal FFS or uninsured service availability for 2-3 weeks depending on implementation timeframe so will have to wait for that.  It is service for 5.5 million people so needs contracting, planning and implementation.

**Comment by coreycaitlin** [comment]
From my email:

This will be a really helpful additional set of resources to point people to. I'm happy to provide the plain language edit, if that can help cut out a step.

Currently on the site, we emphasize that people should always begin by calling their doctor:

Seek treatment by calling your doctor for a phone evaluation if:
you have difficulty breathing (shortness of breath)
you feel like symptoms (such as fever and cough) are getting worse rapidly
you are unable to care for symptoms at home
If you need to go to the hospital, call ahead so they can prepare for your arrival. If you need to call 911, tell the 911 operator you’re experiencing coronavirus symptoms so the ambulance provider can prepare to treat you safely.

I think we can strengthen that answer a bit to further emphasize the importance of using phone/remote care, then add a new question and answer that addresses what people can do if they can't reach their doctor on the phone, or if their doctor is unable to do a phone screening. (That answer will then link to the insurance co's nurse advice lines and other resources.)

Let me know when you have the list of links or numbers for telehealth options people can use if their doctors are unavailable by phone, and I can get this content drafted for review.

**Comment by abquirarte** [comment]
Corey:
Currently on the site, we emphasize that people should always begin by calling their doctor:

Seek treatment by calling your doctor for a phone evaluation if:
you have difficulty breathing (shortness of breath)
you feel like symptoms (such as fever and cough) are getting worse rapidly
you are unable to care for symptoms at home
If you need to go to the hospital, call ahead so they can prepare for your arrival. If you need to call 911, tell the 911 operator you’re experiencing coronavirus symptoms so the ambulance provider can prepare to treat you safely.

I think we can strengthen that answer a bit to further emphasize the importance of using phone/remote care, then add a new question and answer that addresses what people can do if they can't reach their doctor on the phone, or if their doctor is unable to do a phone screening. (That answer will then link to the insurance co's nurse advice lines and other resources.)

Let me know when you have the list of links or numbers for telehealth options people can use if their doctors are unavailable by phone, and I can get this content drafted for review.

**Comment by abquirarte** [comment]
Other suggestions:

**Comment by abquirarte** [comment]
### splash language, Dustin Corcoran <dcorcoran@cmadocs.org

**How Telehealth can Help Doctors and Patients During the COVID-19 Pandemic**
Telehealth allows physicians to stay connected and provide care to patients without an in-person visit by utilizing audio and video technologies. During the unprecedented COVID-19 outbreak, the public health benefits of telehealth have been clear. Telehealth allows patients to receive care in the comfort of their homes, while helping physicians to reduce infection rates and transmission and ease the strain on the delivery system. The Centers for Disease Control and Prevention has recommended that care be delivered virtually during the COVID-19 crisis as the best way to protect patients, physicians and medical staff. 

California and the federal government have issued a series of waivers, guidance, and directives making it easier for physicians to use telehealth to interact with their patients during this crisis.  These changes improve how physicians get reimbursed for telehealth, as well as providing flexibility in privacy laws to allow practices to operate more efficiently.

For patients, telehealth allows you to speak to your own doctor about new or ongoing symptoms, concerns, or conditions, without unnecessary exposure to COVID-19. Patients should consider speaking with their physician virtually, whether those concerns are related to COVID-19 or not. 
Telehealth allows physicians to continue to be the trusted source of care and health information for their patients. Telehealth gives doctors the best option to keep their patients, staff, and themselves protected, while still being able to access and triage their patients’ symptoms and manage ongoing conditions or health issues. 

There are a few different ways to conduct telehealth visits. Some patients will have access to a telehealth option through their patient portals, allowing physician and patients to leverage a tool already built into a portal they may be used to. Some physicians will have access to a telehealth option that is integrated into the electronic health record platform they use.  For many patients and physicians, however, the quickest solution will be to access to standalone solutions. These standalone solutions include vendors that specifically offer telehealth technologies for physicians to continue to see their own patients or standalone medical groups with doctors that can help manage overflow. During the COVID-19 outbreak, these standalone solutions also temporarily include non-public facing remote connections, such as Apple FaceTime, Facebook Messenger video chat, Google Hangouts video, and Skype.

**Comment by abquirarte** [comment]
New version: 
[Telehealth Splash Page Draft 03_25_20 v2_pm edit_VP Edits.docx](https://github.com/cagov/covid19/files/4384734/Telehealth.Splash.Page.Draft.03_25_20.v2_pm.edit_VP.Edits.docx)

**Comment by coreycaitlin** [comment]
I think the shape and design of this page will really depend on the data itself, so we should begin prototyping only once we have at least example data in hand.

Once launched, we'll want to create pathways to this content from the **Health care** and **Testing and treatment** pages, and any time we mention calling a healthcare provider.

**Comment by coreycaitlin** [comment]
NB: It will be framed differently on those 2 pages —

1. On the Testing and treatment page, it'll be framed as "the safest way to get treated for COVID-19 is via telehealth"
2. On the Health care page, it'll _also_ be framed in terms of general access to health care for all issues

**Comment by abquirarte** [comment]
### From Corey 3/26/2020

_Because we're in a crisis, we should assume that:_

- Users are reading even less than usual online, and comprehending less of what they read
- Users are skimming websites for the information most relevant to them
- Users are looking for actions they can take

_The primary audience of this site is people seeking treatment, which means that much of the behind-the-scenes information about billing, reimbursement, privacy, medical records, etc. will be irrelevant to our core users. If we need to address those issues, I'd recommend we cover them in a Q&A section after the primary call to action, which will be "Find your options" or "Get telehealth."_

_The introductory content that frames that call to action should be extremely brief, and we must assume that readers are not familiar with industry terms like "telehealth," "patient portal," or "telehealth vendor." We should choose carefully which ones are worth introducing (I think we should introduce the term "telehealth" but use plain-language alternatives for all others), and keep the rest of the content as concise as possible. Here's an example of the kind of introduction I think would make sense:_

>During this coronavirus outbreak, the safest way to get medical care is over the phone or online. This is called “telehealth” or “telemedicine.” Many doctors and insurance companies offer telehealth options for COVID-19 screening. Telehealth is also the best way to get most other medical care (including other symptoms, mental health care, or ongoing conditions) during the Stay Home Order.
 
>The simplest way to get treated over the phone is to call your doctor. If this isn’t an option (for instance, you don’t have a doctor, your doctor is too busy, or your doctor can’t treat you over the phone), your insurance company may offer other telehealth options. Nurse lines, insurance patient portals (websites), and video consultation are several common kinds of telehealth.
 
>Find your telehealth options by choosing your health care insurance company here:

_Even this brief language still needs further editing for simplicity of sentence structure and flow, but we'll want to save most of that refinement until we have the core data and are able to begin prototyping the interface users will use to access that data._

**Comment by abquirarte** [comment]
We got the spreadsheet (still in the works) @coreycaitlin ! 
>From chhs- we’d mostly pull information from Columns A, F, and G to populate a list of telehealth resources by health plan.

[Telehealth 3.27.20.xlsx](https://github.com/cagov/covid19/files/4398132/Telehealth.3.27.20.xlsx)

**Comment by abquirarte** [comment]
"Final" approved by SME team. 

[Telehealth Splash Page Draft 03_27_20_final clean.docx](https://github.com/cagov/covid19/files/4398138/Telehealth.Splash.Page.Draft.03_27_20_final.clean.docx)

**Comment by abquirarte** [comment]
[screencapture-hemingwayapp-2020-03-31-22_39_22.pdf](https://github.com/cagov/covid19/files/4412985/screencapture-hemingwayapp-2020-03-31-22_39_22.pdf)

**Comment by abquirarte** [comment]
1. Some links go to the INFO about telemedicine for that provider and some take you to the Vendor sign up (that is linked to that provider). Which one should we use? I would think its prob the Info page since that page links to how you can enroll with these vendors 
_I agree with info page -AQ and hyperlink in the name itself?_
So could we have 1 OR 2 links for each provider or is that a headache?
2. The info varies per provider
_i asked about this, show for now-AQ_
3. there are a couple that don't have any info listed...I list a suggestion in the sheet
_i asked about this-list for now_
4. As a team, we should figure out if we need to include any note vs just the link, number if listed and provider name...(That would be my suggestion) 
_I think it depends on the note, if it's helpful context I'd rather give it to them before we point them out to find that out for themselves-AQ_ 


Draft in WP- Titled: [Online and medical care options (telehealth)](https://as-go-covid19-d-001.azurewebsites.net/?p=691&preview=true) - we need a shorter title lol @coreycaitlin 

@aaronhans or @artuoma how could we best represent the data @hilaryhb helped clean up? https://docs.google.com/spreadsheets/d/1w6vMSDb6tYfm5JOSapfQbw0-Digkmbxd/edit#gid=1284817610

**Comment by aaronhans** [comment]
Looks like a candidate for a table layout: https://getbootstrap.com/docs/4.4/content/tables/ I'll defer to @artuoma on design recommendations. We can go live with an initial hardcoded version that has the data in HTML or local json file then to automate:
- [ ] Pull data directly from the spreadsheet using google sheets API
- [ ] Run sheets query every few minutes via FAAS
- [ ] FAAS creates json file of sheet data, stores in blob storage, compares to last version
- [ ] if change writes update to github master and staging branches

**Comment by abquirarte** [comment]
DHCS- new data- I think we are merging?
[Covid-19 Information - Local Plan Landing Pages (1).xlsx](https://github.com/cagov/covid19/files/4416324/Covid-19.Information.-.Local.Plan.Landing.Pages.1.xlsx)

**Comment by artuoma** [comment]
@abquirarte @aaronhans Depends if we are trying to display all the data at once or based on some kid of search or user input like location.

**Comment by abquirarte** [comment]
### Outlined content for the commercial plans:
[Telehealth Resources_4.1.20.docx](https://github.com/cagov/covid19/files/4424743/Telehealth.Resources_4.1.20.docx)

### Notes:
1.       Collapsible Content: I am thinking that we can have each health plan listed alphabetically with collapsible sections to show the details. This way if some information is variable, that’s okay. For example, some provide a nurse advice line and some don’t. I’m going to circle back with CAHP to ask them if we’d like to have more consistency in nurse advice line information.

2.       Medi-Cal: It’s unclear if all the commercial plans participating in Medi-Cal (e.g., Blue Shield, Health Net) also provide telehealth services to Medi-Cal members. Molina explicitly stated they provide access to Teledoc for Marketplace (Covered California) and Medicare Advantage members, but not yet for Medi-Cal. This is also an evolving issue with the new guidance.

3.       Nurse Advice Lines: We may want to explain what this is on the landing page. It’s generally a free service, while telehealth could be charged as a visit. I’ll take a look at your mockup to see where that could go.

4.       Variation in Cost-Sharing: It’s uneven among the health plans whether they waive cost-sharing. I am thinking we shouldn’t include this information because we’d have to keep updating it if it changes. We probably should just be a central repository of links/resources and not the source for cost-sharing information.

5.       How to integrate Medi-Cal local health plans: Dr. Gilbert said to link to the homepage of each local health plan, which includes telehealth resources upfront. I still need to individual go through each one to see what is available. A local health plan is a particular model in Medi-Cal that is very dependent on county of residence. Not sure we need to accept as is from DHCS in their table format or figure out a way to incorporate into the list we already have.

6.       Medi-Cal FFS/Uninsured: DHCS is finalizing the contract this week so we should have a sense early next week of what this resource is and how to incorporate it.

### ALL spreadsheets here https://docs.google.com/spreadsheets/d/1w6vMSDb6tYfm5JOSapfQbw0-Digkmbxd/edit#gid=1067488049

**Comment by aaronhans** [comment]
Made @artuoma's prototype dynamic here: https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net/telehealth/ it combines the content from the WordPress powered telehealth post and a list of providers that is generated from a json file in pages/_data/telehealth.json

We can update this json when we have a clean enough dataset to go live and the depending on how often this data will get updated either:
- publish changes from the spreadsheet into github like we do with news articles
or
- pull the provider data from a new API

I opened a new ticket https://github.com/cagov/covid19/issues/181 to upgrade our APIs to be able to handle massive traffic that might be generated by the covid site without incurring too much cost from unnecessary FAAS instances firing up. Our current non cached endpoints will scale automatically but adding a caching or publish to CDN backed blog storage layer makes everything faster and cheaper.

**Comment by aaronhans** [comment]
https://github.com/cagov/covid19/commit/077e3dca421bbfecfec08a61ffc85fff2b827a34

**Comment by artuoma** [comment]
SME is working on a comprehensive [spreadsheet](https://docs.google.com/spreadsheets/d/1w6vMSDb6tYfm5JOSapfQbw0-Digkmbxd/edit#gid=1067488049) that will be used on the [telehealth page](https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net/telehealth/).

**Comment by abquirarte** [comment]
> **Status update:**
> SME is working on a comprehensive [spreadsheet](https://docs.google.com/spreadsheets/d/1w6vMSDb6tYfm5JOSapfQbw0-Digkmbxd/edit#gid=1067488049) that will be used on the [telehealth page](https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net/telehealth/).

I heard from him earlier, we was going to take a stab at the spreadsheet at 4

**Comment by abquirarte** [comment]
[Staged content is in this doc](https://docs.google.com/document/d/1I3bAyVF6QD1BEMSmf9aH0NYWCE71BpC39bFw4_pcYXg/edit#)
![telehealth-04072020](https://user-images.githubusercontent.com/9556188/78732753-5e726800-78f8-11ea-9f69-a91fa5656113.PNG)

**Comment by aaronhans** [comment]
Latest data is in staging along with all data related updates https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net//telehealth/

**Comment by abquirarte** [comment]
- [ ] health plans be alphabetized when the results show up?

- [ ] For some reason, it's defaulting to "No" for all listings for "Telehealth care offered." I put in zip code 94530.  For example, Kaiser should be yes. 

- [ ]  this language should refer to health plans instead of providers:

- [ ] Original: 94530 is in Contra Costa County, showing providers in Contra Costa County

- [ ] Update to: 94530 is in Contra Costa County, showing telehealth services or nurse advice lines offered by health plans in Contra Costa County.

**Comment by thepegisin** [comment]
92102 is in San Diego County, showing telehealth services or nurse advice lines offered by health plans in San Diego County.

^ This text is quite a mouthful. This would be better:

Showing health plan telehealth options for 92102: San Diego County.

**Comment by thepegisin** [comment]
Or even just:

Showing telehealth options for 92102: San Diego County.

**Comment by abquirarte** [comment]
Aaron sent back to Vishaal for review.

**Comment by abquirarte** [comment]
Sponsor sent in email. Suggested final content edits pasted below.  

If you’re having a medical emergency, you should call 911 or go to the nearest emergency room.
During this coronavirus (COVID-19) outbreak, the safest way to get medical care is by phone or video while staying at home. This is called “telehealth.”
Telehealth is the first step in getting medical care including:

- •	If you have coronavirus symptoms and think you need testing or treatment. Many health care providers and health plans offer telehealth options for COVID-19 screening.
- •	If you need treatment for symptoms of other conditions, mental health care, substance abuse treatment, or follow-up care to treat ongoing conditions during the stay home order.
- How to get telehealth:
- •	The simplest way to get treated over the phone is to call your regular health care provider. 
- •	If your doctor isn’t available, your health plan may offer other telehealth options through their member websites. If telehealth isn’t available from your doctor or your health plan, you can consult nurse advice lines.

Find telehealth or nurse advice line options offered by your health plan by entering your county or zip code here:

You should not have to pay more for telehealth than you would for an in-person visit. Some health plans or providers may even waive any copay.
If you have a hearing or speech disability, you can dial 711 to be connected to a free telecommunications relay service which can be used with a text telephone (TTY) or other device.  
If you have trouble accessing your regular health care provider through phone or telehealth, here’s what you can do:
For private health plans, call the state Department of Managed Health Care’s Help Center at 1-888-466-2219. 
For Medi-Cal, call the state Department of Health Care Service’s Help Center at 1-800-541-5555.

**Comment by aaronhans** [comment]
We would have to put the search box HTML in WP to mix it into the post but will still work

**Comment by aaronhans** [comment]
Moving this to done per Angie. All edits are made, if we get new data we will update immediately, page is browseable at /telehealth but not yet linked from anywhere yet. Latest word is announcement is planned 5/1
